src/en/comicextra/src/eu/kanade/tachiyomi/extension/en/comicextra/ComicExtra.kt

```diff
@@ -40,7 +40,7 @@ class ComicExtra : ParsedHttpSource() {
 
     override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
         return if (query.isNotBlank()) {
-            GET("$baseUrl/comic-search?key=$query", headers)
+            GET("$baseUrl/search?keyword=$query", headers)
```

Comment:
Use `addQueryParameter` so the query can contain special characters.
Should also have pagination

Reply:
I have made the requested changes

Reply:
CI failed, you also need the imports

Reply:
Hmm it's still failing. One moment let me see what's up, I've tested the change before committing I must have removed something by mistake